client/components/marketing-survey/gsuite-cancel-purchase-dialog/index.jsx

```diff
@@ -235,11 +240,15 @@ GSuiteCancelPurchaseDialog.propTypes = {
 };
 
 export default connect(
-	( state, { purchase } ) => {
+	( state, { purchase, site, domain } ) => {
 		return {
-			productName: getName( purchase ),
 			domain: purchase.meta,
+			productName: getName( purchase ),
 			purchasesError: getPurchasesError( state ),
+			selectedDomain: getSelectedDomain( {
+				domains: getDomainsBySiteId( state, site.ID ),
+				selectedDomainName: domain,
```

Comment:
`domain` is undefined here, because we're deriving domain from `purchase.meta` on line 245. We should fix this line and remove `domain` from line 243 as well.
```suggestion
				selectedDomainName: purchase.meta,
```